blocks/large-manual-promo-block/_index.scss

```diff
@@ -0,0 +1,11 @@
+@use "@wpmedia/arc-themes-components/scss";
+
+.b-large-manual-promo {
+	&__info {
+		@include scss.block-properties("large-manual-promo-info");
+		@include scss.block-components("large-manual-promo-info");
+	}
```

Comment:
```suggestion
```

is this being used anywhere? if not, like the previous classes that were removed, this can be removed as well

Reply:
oh I missed it.